modules/rhel-preparing-node.adoc

```diff
@@ -70,3 +70,10 @@ Note that this might take a few minutes if you have a large number of available
     --enable="rhel-7-server-extras-rpms" \
     --enable="rhel-7-server-ose-4.1-rpms"
 ----
+
+. If firewalld has been started on the host, disable and stop the service:
++
+----
+# systemctl disable firewalld.service
+# systemctl stop firewalld.service
+----
```

Comment:
Is firewalld enabled by default? If so, I might say something like
```
. Stop and disable firewalld on the host:
+
----
# systemctl disable firewalld.service
# systemctl stop firewalld.service
----
+
[NOTE]
====
You must not enable firewalld later. If you do, you cannot access {product-title} logs on the worker.
====
```

@mtnbikenc, what do you think?

Reply:
We should also mention the scaleup and upgrade playbooks will ensure firewalld is disabled.

Reply:
Are you backporting this change to 4.1?

Reply:
@kalexand-rh I've updated both scaleup and upgrade playbooks sections it's mentioned.